in_toto/verifylib.py

```diff
@@ -1568,6 +1571,9 @@ def in_toto_verify(layout, layout_key_dict, link_dir_path=".",
               - artifact rules in step and inspection definitions in the layout
               - the run fields in the inspection definitions
               - the expected command in the step definitions
+    step_name: (optional)
+            The step that the layout corresponds to, used only for sublayout
+            verification to generate summary links.
```

Comment:
Maybe it is worth pointing out that the parameter is used in recursive calls of `in_toto_verify` and to refer the reader to `verify_sublayouts` to see how it is used?